admin/include/backup.php

```diff
@@ -3,173 +3,171 @@
 // *** feb. 2023: Rebuild this script by Huub. Multiple backups will be stored on server. ***
 
 @set_time_limit(3000);
-@ini_set('memory_limit','-1');
+@ini_set('memory_limit', '-1');
 error_reporting(E_ALL);
 // *** Safety line ***
-if (!defined('ADMIN_PAGE')){ exit; }
+if (!defined('ADMIN_PAGE')) {
+	exit;
+}
 
 // *** Move and remove files from previous backup procedure ***
-if (file_exists('humo_backup.sql.zip')){
-	$new_file_name='backup_files/'.date("Y_m_d_H_i",filemtime('humo_backup.sql.zip')).'_humo-genealogy_backup.sql';
-	rename('humo_backup.sql.zip',$new_file_name);
+if (file_exists('humo_backup.sql.zip')) {
+	$new_file_name = 'backup_files/' . date("Y_m_d_H_i", filemtime('humo_backup.sql.zip')) . '_humo-genealogy_backup.sql';
+	rename('humo_backup.sql.zip', $new_file_name);
 
-	if (file_exists('downloadbk.php')){
+	if (file_exists('downloadbk.php')) {
 		unlink('downloadbk.php');
 	}
 }
-if (file_exists('backup_tmp/readme.txt')){
+if (file_exists('backup_tmp/readme.txt')) {
 	unlink('backup_tmp/readme.txt');
 	rmdir('backup_tmp');
 }
 
 echo '<h1 class="center">';
-	printf(__('%s backup'),'HuMo-genealogy');
```

Comment:
The identation is removed. But I think the code is better readible if this identation isn't removed.